src/openforms/logging/logevent.py

```diff
@@ -441,18 +449,94 @@ def submission_export_list(form: "Form", user: "User"):
 def submission_logic_evaluated(
     submission: "Submission", evaluated_rules, updated_submission_data
 ):
```

Comment:
I'd rename the variables here to be a bit more generic - now the mechanisms of the actual logic evaluation are leaking into this log function, but that's not really relevant.

```suggestion
def submission_logic_evaluated(
    submission: "Submission", evaluated_rules, resulting_data
):
```

I am wondering though - the startpoint data is essentially `submission.data`, but as part of the logic evaluation, this data can change values. This masks the actual values that are used for evaluation of a particular rule. There is a delta possible between `submission.data` and `resulting_data` - even for the same key!